packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -131,6 +131,12 @@
           "group": "navigation"
         }
       ],
+      "editor/title/context": [
+        {
+          "command": "sfdx.force.lightning.lwc.preview",
+          "when": "sfdx:project_opened && resource =~ /.*/lwc/[^/]+(/[^/]+\\.(html|css|js))?$/"
```

Comment:
Out of curiosity has this been tested on a windows machine? I'm having all problems with the extension after upgrading (#1832). 

A long shot maybe, but I've seen subtle bugs on windows caused by unquoted/unescaped asterisk wildcards in package.json regex patterns before